cli/src/commands/log.rs

```diff
@@ -38,8 +38,9 @@ use crate::ui::Ui;
 /// rendered as a synthetic node labeled "(elided revisions)".
 #[derive(clap::Args, Clone, Debug)]
 pub(crate) struct LogArgs {
-    /// Which revisions to show. Defaults to the `revsets.log` setting, or
-    /// `@ | ancestors(immutable_heads().., 2) | trunk()` if it is not set.
+    /// Which revisions to show. If no paths are specified, this defaults to the
```

Comment:
nit: "no paths nor revisions" for clarity?

Reply:
agree, that sounds better